readme-tw-gtasks.md

```diff
@@ -0,0 +1,87 @@
+# [Taskwarrior](https://taskwarrior.org/) ⬄ [Google Tasks](https://support.google.com/tasks/answer/7675772?hl=en&co=GENIE.Platform%3DDesktop)
+
+## Description
+
+Given all tasks in your Google Task task list of a
```

Comment:
minor corrections: Given all tasks in your Google Tasks *and the* task list of a taskwarrior _filter_ ...